### ISSUE TITLE
No randomization of subject id and primary key, and each value at least once in fake data generation 

### ISSUE THREAD

**Issue by EmV25** [OHDSI/WhiteRabbit#201]
When generating fake data, White Rabbit takes the frequency of the values in the scan report into account. This results in absence of low frequency values in the fake data, while these values are of importance when implementing and running the ETL. A solution for this would be to store every value in the scan report at least once in the fake data tables.

Furthermore, fields in the scan reports with seemingly random numbers are not used in the fake data as is, but other random numbers are added. This could be an issue when dealing with primary keys and person/subject ids. When the scan report is based on multiple tables, and primary keys & subject ids start to mismatch between these tables, this could create issues when creating the ETL.
A solution for this issue would be the ability to assign fields as either primary key or subject id before generating the fake data, use these values as they are in the scan report, and not randomize the values in these fields in the fake data.

In addition, it would be an advantage if for each scanned table you could indicate how many rows of fake data you want to generate. So for example for table A a 1000 rows are generated, and for table B 10000 rows.

 To summarize:
- [ ] Store every value in the scan report at least once in the fake data
- [ ] Assign primary key and subject id fields, and do not randomize these fields. Take values as is.
- [ ] Ability to create different number of rows per table

- [ ]

**Comment by MaximMoinat** [comment]
Pull request #203 fixes the first two issues

Still to be done:
- [ ] generation of fake date and fake real number for 'random' generation (if scanned values not available)
- [ ] implement a foreign key generation, and/or re-implement an improved primary key generation (see #184)

Tagging issue #217 as well
